compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1075,6 +1076,16 @@ rustc_queries! {
         cache_on_disk_if { true }
     }
 
+    /// Evaluate a static's initializer, returning the allocation of the initializer's memory.
+    query eval_static_initializer(key: DefId) -> EvalStaticInitializerRawResult<'tcx> {
+        desc { |tcx|
+            "evaluating initializer of static `{}`",
+            tcx.def_path_str(key)
+        }
+        cache_on_disk_if { key.is_local() }
+        separate_provide_extern
```

Comment:
Same here, this might need review by someone else. I have no idea about the tradeoffs involved in configuring a query.